recipe/meta.yaml

```diff
@@ -143,3 +138,8 @@ extra:
   recipe-maintainers:
     - simonbray
     - jaimergp
+    - dacase
+    - mikemhenry
+    - mattwthompson
+    - j-wags
+    - swails
```

Comment:
@dacase @mikemhenry @j-wags do you consent?

Reply:
Yes please!

Reply:
I agree to be a maintainer!